### DISCUSSION TITLE
here to seek guidance over TOWFIG+NOODL+BECOS

### DISCUSSION THREAD

**Discussion by Jibo110** [community/community#122041] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hi All,

My cliet has asked me to look into TOWFIG+NOODL+BECOS, now i have looked into Noodl but couldn't dfind other 2, there is possiblity i am searching wroge or whatsoever. Can anyone guide me with these 3.

Thank you

**Comment by ebndev** [comment]
Hey @Jibo110,

Thanks for being a part of the GitHub Community!

Maybe it would be best to ask your client to clarify what they mean. Although other users may have encountered this, it would be best for you to know exactly what you need to be searching in case you don't get a response here as quick as you need to.

**Reply by Jibo110** [reply]
Hi @ebndev 

Thanks for your response, my client asked me if i knew TOWFIG + NOODL + BECOS. I said to him that i'll look into it. Now i am trying to search for these online but could'nt find anything else except for NOODL. My cleint wants software and other projects on Python, swift, Django, Javascript and Electron js. So now i don't know what are these tech relate to, and honestly don't know if they are tech or what. Noodle is like webflow, bubble.io. 

If you can help me with that. Thanks in advance

**Reply by ebndev** [reply]
I'm not familiar with those either but I'll move this post to our Programming Help category as it may get more relevant help there!

**Reply by Jibo110** [reply]
Thank you
